client/components/team/TeamListCard.jsx

```diff
@@ -116,9 +116,8 @@ TeamListCard = class TeamListCard extends Component {
       <Header as='h4'>
         <Header.Content>
           { name }
-          <Header.Subheader>
-            <p><Icon name='mail' size='small'/><a href={`mailto:${email}`}>{email}</a></p>
-            <p><Icon name='phone' size='small'/><a href={`tel:${phone}`}>{phone}</a></p>
+            <Header.Subheader>
```

Comment:
Potential follow-on PR: could we make "matchmaking" an admin-toggle-able feature? If we find a serious issue with it and want to turn it off without doing a code push, I'd love to be able to hit a button in the admin screen. My thinking is that `gameplay.matchmaking_enabled` or whatever it'd be called would both (a) show/hide this button and (b) potentially short-circuit the `user.message` call on the server

Reply:
Good idea, but not doable in the time available before we plan to start registration this year. I'll make a separate issue for this.